styles.css

```diff
@@ -112,3 +72,359 @@ h1 {
   gap: 20px;
   align-items: center;
 }
+
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.snapshot {
+  height: 220px;
+  width: 295px;
+  left: 16px;
+  top: 16px;
+  border-radius: 8px;
+}
+
+.tonic {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.stack {
+  display: flex;
+  justify-content: left;
+  gap: 5px;
+  list-style: none;
+}
+
+.stack-elem:hover {
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  color: #fff;
+  border: #6070ff;
+}
+
+.stack-elem:active {
+  background: #2230d2;
+}
+
+.stack .stack-elem {
+  width: 33px;
+  height: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  background-color: #ebebff;
+  border-radius: 8px;
+  border: none;
+}
+
+.About-myself {
+  background-color: #fff;
+  margin: 114px 24px;
+  display: flex;
+  flex-direction: column;
+  height: 1263px;
```

Comment:
- [ ] Nice work so far but I kindly suggest you remove the height style from the About section as it tends to add unnecessary white space below your application